joss_paper/paper.md

```diff
@@ -0,0 +1,221 @@
+
+![RMSD benchmarks performed on the same machine as \autoref{fig:benchmark}. YiiP trajectory aligned to first frame as reference using `MDAnalysis.analysis.align.AlignTraj` and converted to compressed, quantized H5MD (7.8GB) and ZarrMD (4.9GB) trajectories. RMSD performed using development branch of MDAnalysis (2.8.0dev) with "serial" and "dask" backends. See [this notebook](https://github.com/Becksteinlab/zarrtraj/blob/d4ab7710ec63813750d7224fe09bf5843e513570/joss_paper/figure_2.ipynb) for full benchmark codes. \label{fig:RMSD}](RMSD.png)
+
+*Zarrtraj* is capable of making use of *Zarr*'s powerful compression and quantization when writing ZarrMD trajectories. 
+The uncompressed MDAnalysisData YiiP trajectory in ZarrMD format is reduced from 11.3GB uncompressed 
+to just 4.9GB after compression with the Zstandard algorithm [@Zstandard:2021] 
+and quantization to 3 digits of precision. See [performance considerations](https://zarrtraj.readthedocs.io/en/latest/performance_considerations.html)
+for more.
+
+# Example
+
+The YiiP membrane protein trajectory [@YiiP:2019] used for benchmarking in this
+paper is publicly available for streaming from the Google Cloud Bucket
+*gcs://zarrtraj-test-data/yiip.zarrmd*. The topology file in PDB format, which contains
+information about the chemical composition of the system, can also be accessed
+remotely from the same bucket (*gcs://zarrtraj-test-data/YiiP_system.pdb*) using
+[fsspec](https://filesystem-spec.readthedocs.io/en/latest/), although this is
+currently an experimental feature and details may change.
+
+In the following example (see also the [YiiP Example in the zarrtraj
+docs](https://zarrtraj.readthedocs.io/en/latest/yiip_example.html)), we access
+the topology file and the trajectory from the *gcs://zarrtraj-test-data* cloud
+bucket. We initially create an `MDAnalysis.Universe`, the basic object in
+MDAnalysis that ties static topology data and dynamic trajectory data together
+and manages access to all data. We iterate through a slice of the trajectory,
+starting from frame index 100 and skipping forward in steps of 20 frames:
+
+```python
+import zarrtraj
+import MDAnalysis as mda
+import fsspec
+
+with fsspec.open("gcs://zarrtraj-test-data/YiiP_system.pdb", "r") as top:
+    u = mda.Universe(top, "gcs://zarrtraj-test-data/yiip.zarrmd", 
+	                 topology_format="PDB")
+
+    for ts in u.trajectory[100::20]:
+        print(ts)
+```
+
+Inside the loop over trajectory frames we print information for the current
+frame `ts` although in principle, any kind of analysis code can run here and
```

Comment:
It’s unclear to others what ts refers to. Better explicitly mention that it stands for `timestep` and contains all the frame-wise trajectory data.

Reply:
Yeah might be better to use `for timestep in u.trajectoru[100::20]:` in the above example